content/blog/2023-09-27-epidatr.Rmd

```diff
@@ -0,0 +1,103 @@
+---
+title: New package released! Epidatr, an R client for Delphi Epidata API
+author: Dmitry Shemetov, David Weber
+date: 2023-09-27
+tags:
+  - r
+  - epidata
+  - covidcast
+authors:
+  - dmitry
+  - davidweb
+heroImage: blog-lg-epidatr.jpg
+heroImageThumb: blog-thumb-epidatr.jpg
+summary: |
+  `epidatr` is designed to streamline the downloading and usage of data from the [Delphi Epidata API](https://cmu-delphi.github.io/delphi-epidata/). It provides a simple R interface to the API, including functions for downloading data, parsing the results, and converting the data into a tidy format.
+
+output:
+  blogdown::html_page:
+    toc: true
+---
+
+The [Delphi Epidata API](https://cmu-delphi.github.io/delphi-epidata/) provides real-time access to epidemiological surveillance data for influenza, COVID-19, and other diseases from both official government sources such as the [Center for Disease Control (CDC)](https://www.cdc.gov/datastatistics/index.html), private partners such as [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Change Healthcare](https://www.changehealthcare.com/), and other public datasets like [Google Trends](https://console.cloud.google.com/marketplace/product/bigquery-public-datasets/covid19-search-trends). It is built and maintained by the Carnegie Mellon University [Delphi research group](https://delphi.cmu.edu/).
```

Comment:
```suggestion
The [Delphi Epidata API](https://cmu-delphi.github.io/delphi-epidata/) provides real-time access to epidemiological surveillance data for influenza, COVID-19, and other diseases from both official government sources such as the [Centers for Disease Control and Prevention (CDC)](https://www.cdc.gov/datastatistics/index.html), private partners such as [Meta](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Change Healthcare](https://www.changehealthcare.com/), and other public datasets like [Google Trends](https://console.cloud.google.com/marketplace/product/bigquery-public-datasets/covid19-search-trends). It is built and maintained by the Carnegie Mellon University [Delphi Research Group](https://delphi.cmu.edu/).
```

Additionally, Change Healthcare has merged into Optum, but "Change Healthcare" might still count as a "current" name, based on [https://www.changehealthcare.com/](https://www.changehealthcare.com/).  I didn't include any change in the suggestion above, since it's probably easier for readers to search the API docs for Change and get the right endpoint.

Reply:
These make sense, though i might say something like "Facebook (now Meta)" since they were Facebook when we partnered with them and because we refer to them as such in the linked article.